src/client/sandbox/event/selection.js

```diff
@@ -87,8 +88,10 @@ export default class Selection {
                 return selectionSetter();
             }
 
-            // NOTE: setSelectionRange leads to focusing an element only in IE.
-            return selection.wrapSetterSelection(el, selectionSetter, browserUtils.isIE && browserUtils.version < 12);
+            const needFocus = browserUtils.isIE && browserUtils.version < 12 ||
```

Comment:
Simplify to `const needFocus = browserUtils.isIE11 || browserUtils.isMSEdge`

Reply:
We decide to leave MSEdge version. But you can simplify `browserUtils.isIE && browserUtils.version < 12` to `browserUtils.isIE11`